src/main/provisioning/repoinit.txt

```diff
@@ -71,6 +71,13 @@
         allow	rep:write	on /apps/sling/install
     end
 
+    # content-package installer
+    create service user sling-package-install
+
+    set ACL for sling-package-install
+        allow	jcr:all     on	/
```

Comment:
seems a bit too broad, especially as right now the service user is only used to upload packages: https://github.com/apache/sling-org-apache-sling-installer-factory-packages/blob/84b0f9243e2c7cc733762e8ce199549632eaf9a2/src/main/java/org/apache/sling/installer/factory/packages/impl/PackageTransformer.java#L146. The actual installation still takes an admin session (https://github.com/apache/sling-org-apache-sling-installer-factory-packages/blob/84b0f9243e2c7cc733762e8ce199549632eaf9a2/src/main/java/org/apache/sling/installer/factory/packages/impl/PackageTransformer.java#L262), this should be fixes though as well.

Reply:
After having a 2nd thought I think this should be fine as once we fixed https://issues.apache.org/jira/browse/SLING-9422 we definitely need to be install all packages (which might everywhere to the repo)

Reply:
Ack, thanks. I'll leave that in then, as we should get a release of the installer.factory.packages bundle at some point.